checkers/rangeAppendAll_checker.go

```diff
@@ -0,0 +1,102 @@
+package checkers
+
+import (
+	"go/ast"
+	"go/token"
+
+	"github.com/go-critic/go-critic/checkers/internal/astwalk"
+	"github.com/go-critic/go-critic/linter"
+	"github.com/go-toolsmith/astcast"
+	"golang.org/x/tools/go/ast/astutil"
+)
+
+func init() {
+	var info linter.CheckerInfo
+	info.Name = "rangeAppendAll"
+	info.Tags = []string{linter.DiagnosticTag, linter.ExperimentalTag}
+	info.Summary = "Detects append all its data while range it"
+	info.Summary = "Detects suspicious sort.Slice calls"
```

Comment:
Perhaps this line should be removed.

Reply:
stupid copy and paste, maybe someday we can add an `copyandpaste` lint 😂 , I will remove it later.